### PR TITLE
Pr/9423

### PR DESCRIPTION
**Pull request by hamiltonha**

### CODE REVIEW

**Comment by jasonjoh** [comment]
Please disregard the earlier message from msftbot. Your pull request was incorrectly flagged.

File: api-reference/v1.0/api/application-update.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ In the request body, supply the values for relevant fields that should be update
...
 | web                     | [webApplication](../resources/webapplication.md)                            | Specifies settings for a web application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| spa                     | [spaApplication](../resources/spaapplication.md)                                         | Specifies settings for a single-page application. `spa` type redirect URIs notify the server to enable CORS and use the authorization code flow.                  |
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/application-update.md:62]: Is this addition for beta or both beta and v1? If the addition applies to both, please include the changes in beta as well.

File: api-reference/v1.0/resources/spaapplication.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "spaApplication resource type"
+description: "Specifies settings for a spa application."
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/spaapplication.md:3]: The description in line 3 should match the one in line 14. In this case, I'd appreciate if you use "single-page application" in both lines.

File: changelog/Microsoft.DirectoryServices.json [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "da2e0aa4-fcae-4c05-8e30-e673ec56db9d",
+          "ApiChange": "Resource",
+          "ChangedApiName": "Update application",
+          "ChangeType": "Addition",
+          "Description": "Introduced the [Single Page Application type]((https://docs.microsoft.com/en-us/graph/api/resources/spaaplication.md)",
+          "Target": "updateApplication"
```
Comment (isvargasmsft) [inline changelog/Microsoft.DirectoryServices.json:11]: Please see the guidelines here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog?anchor=changelist-record-properties

ChangedApiName should be spaApplication. 
Description: Added the [spaApplication] (please check the typo in the link: "spa**apl**ication") resource type.
Target: spaApplication.

Typo in link: there is a 'p' missing.

File: api-reference/v1.0/resources/application.md [hunk 4]
```diff
@@ -83,6 +83,7 @@ This resource supports using [delta query](/graph/delta-query-overview) to track
 | tokenEncryptionKeyId |String|Specifies the keyId of a public key from the keyCredentials collection. When configured, Azure AD encrypts all the tokens it emits by using the key this property points to. The application code that receives the encrypted token must use the matching private key to decrypt the token before it can be used for the signed-in user.|
 | verifiedPublisher          | [verifiedPublisher](verifiedPublisher.md)                            | Specifies the verified publisher of the application.|
 | web |[webApplication](webapplication.md)| Specifies settings for a web application. |
+| spa |[spaApplication](spaapplication.md)| Specifies settings for a single-page application. `spa` type redirect URIs notify the server to enable CORS and use the authorization code flow. |
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/application.md:86]: In beta, the description is: "Specifies settings for a single-page application, including sign out URLs and redirect URIs for authorization codes and access tokens."

Can we use this description for consistency?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @hamiltonha, 

I noticed the changes you propose in this PR are already live in both beta and v1. 

Could you please confirm? If so, may I close this PR?

Kind regards, 

Isaac Vargas

**Comment by isvargasmsft** [comment]
Author confirmed on the side that these changes are already live and this PR can be closed.

**Pull request closed without merging**